packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/Characteristics/components/FilterButton.spec.js

```diff
@@ -0,0 +1,70 @@
+import { mount } from 'enzyme'
+import { Grommet } from 'grommet'
+import React from 'react'
+import sinon from 'sinon'
+import zooTheme from '@zooniverse/grommet-theme'
+import { CloseButton, Media } from '@zooniverse/react-components'
+
+import FilterButton, { StyledFilter } from './FilterButton'
+
+describe('Component > FilterButton', function () {
+  let wrapper, onFilterSpy
+
+  before(function () {
+    onFilterSpy = sinon.spy()
+    wrapper = mount(
+      <FilterButton
+        characteristicId='LK'
+        checked={false}
+        onFilter={onFilterSpy}
+        valueImageSrc='bird-icon.svg'
+        valueLabel='bird'
+      />, {
+        wrappingComponent: Grommet,
+        wrappingComponentProps: { theme: zooTheme }
+      }
+    )
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok()
+  })
+
+  it('should render the characteristic value media', function () {
+    expect(wrapper.find(Media)).to.have.lengthOf(1)
+    expect(wrapper.find(Media).props().src).to.equal('bird-icon.svg')
+  })
+
+  it('should render the media with an alt of the value label', function () {
+    expect(wrapper.find(Media).props().alt).to.equal('bird')
+  })
+
+  it('should have a background color neutral-6', function () {
+    expect(wrapper.find(StyledFilter).props().background.color).to.equal('neutral-6')
+  })
+
+  it('should not render a CloseButton', function () {
+    expect(wrapper.find(CloseButton)).to.have.lengthOf(0)
+  })
+
+  describe('when checked', function () {
+    before(function () {
+      wrapper.setProps({ checked: true })
+    })
+    
+    it('should have a background color of accent-2', function () {
+      expect(wrapper.find(StyledFilter).props().background.color).to.equal('accent-2')
+    })
+
+    it.skip('should show a CloseButton on focus', function () {
```

Comment:
unfortunately I wasn't able to get a combination of `html()` and `focus()` to work within a small time box. open to suggestion, or I think it's ok to remove this and the related `hover` tests?

Reply:
Ok to skip testing this for now. I think we'll have to investigate snapshot tools again soon.